packages/jam/state/disputes.ts

```diff
@@ -31,6 +36,11 @@ export class DisputesRecords {
     return new DisputesRecords(goodSet, badSet, wonkySet, punishSet);
   }
 
+  private readonly goodSetDict: ImmutableHashDictionary<WorkReportHash, true>;
+  private readonly badSetDict: ImmutableHashDictionary<WorkReportHash, true>;
+  private readonly wonkySetDict: ImmutableHashDictionary<WorkReportHash, true>;
+  private readonly punishSetDict: ImmutableHashDictionary<Ed25519Key, true>;
```

Comment:
Didn't we have something like `HashSet<Ed25519Key>` to avoid the need of the second generic argument?